grails-app/controllers/org/pih/warehouse/api/StockMovementApiController.groovy

```diff
@@ -827,8 +827,12 @@ class StockMovementApiController {
 
     // Function for getting appropriate filter options based on current list and supporting requests approval
     List<RequisitionStatus> getOutboundRequisitionStatusCodes(Boolean isApprovalRequired, Boolean isElectronicType) {
-        // If a location doesn't have approval required, return listOutboundOptions no matter what list is displayed
+        // If a location doesn't have approval required, return listOutboundOptions when we are on outbound list
+        // but if we are on the requests list return listRequestsOptions
         if (!isApprovalRequired) {
+            if (isElectronicType) {
+                return RequisitionStatus.listRequestOptions()
+            }
             return RequisitionStatus.listOutboundOptions()
```

Comment:
I think this would look better as a one liner:
`return isElectronicType ? RequisitionStatus.listRequestOptions() : RequisitionStatus.listOutboundOptions()`